src/protagonist.h

```diff
@@ -48,7 +48,7 @@ namespace protagonist {
 
         // Wraps snowcrash::Warnings and snowcrash:Blueprint into report object
         // Note: snowcrash::Result::Error is being sent separately as Error object
-        static v8::Local<v8::Object> WrapResult(const snowcrash::ParseResult<snowcrash::Blueprint>& parseResult,
+        static v8::Local<v8::Object> WrapResult(snowcrash::ParseResult<snowcrash::Blueprint>& parseResult,
```

Comment:
Do you want me to change the order of arguments? I wanted to do it but left it as it is because I `(parseResult, options, astType)` is a standard order we are using in drafter API.